dashboard-frontend/src/pages/ConnectPage/ConnectedAccountsPage.tsx

```diff
@@ -0,0 +1,19 @@
+import PageLayout from '../../components/PageLayout';
+import ConnectedAcccountsList from './ConnectAccountsList';
+
+const accounts = [
+  { id: '1', name: 'Fly', slug: 'fly', authType: 'secret' },
```

Comment:
I like it. Probably don't need a separate "id" - the name is unique, no?

Reply:
added `id` back in because TyepScript was throwing an error, I think because Mirage automatically adds an id property to the response.